packages/core/components/DraggableComponent/index.tsx

```diff
@@ -86,6 +89,22 @@ export const DraggableComponent = ({
     }
   }, []);
 
+  const loadedOverrides = useLoadedOverrides({
+    overrides: overrides,
+    plugins: plugins,
+  });
+
+  const DefaultActionBar = () => (
+    <ActionBar label={label}>
+      <DefaultOverride />
+    </ActionBar>
+  );
+
+  const CustomActionBar = useMemo(
+    () => loadedOverrides.actionBar || DefaultActionBar,
+    [loadedOverrides]
+  );
+
```

Comment:
Could you add an example to the `custom-ui` implementation? It's useful for both reviewing, but also checking this functionality builds

Reply:
Done! I've added a debug info action